README.md

```diff
@@ -1,31 +1,158 @@
+    cd ~/thirdparties/
 
-It is the main package, please see its [README][mRD].
+    git clone https://github.com/jbeder/yaml-cpp
+    cd yaml-cpp/
+    mkdir build
+    cd build
+    cmake -DYAML_BUILD_SHARED_LIBS=ON ..
+    make
+    sudo make install
 
-Here you may read [installation instructions][inst].
+# Install ZeroMQ
 
+Install ZeroMQ core engine in C++
+-----------------------------------
+    cd ~/thirdparties/
+    git clone https://github.com/zeromq/libzmq.git --branch latest_release
+    cd libzmq/
+    mkdir build
+    cd build
+    cmake ..
+    make
+    make test
+    sudo make install
 
-utils package
+Install ZeroMQ for C++
+-------------------------
+    cd ~/thirdparties/
+    git clone https://github.com/zeromq/cppzmq.git
+    cd cppzmq/
+    git checkout v4.5.0
+    mkdir build
+    cd build
+    cmake ..
+    make
+    make test
+    sudo make install
+
+
+Install ZeroMQ for C
 ----------------------
+    cd ~/thirdparties/
+    git clone git://github.com/zeromq/czmq.git
+    cd czmq
+    mkdir build
+    cd build
+    cmake ..
+    sudo make install
+
+Install googletest
+------------------
+    cd ~/thirdparties/
+    wget https://github.com/google/googletest/archive/release-1.10.0.zip
+    unzip release-1.10.0.zip
+    rm release-1.10.0.zip
+    mv googletest-release-1.10.0/ googletest
+
+Install openCV
+--------------
+    wget https://github.com/opencv/opencv/archive/3.4.1.tar.gz
+    tar zxvf 3.4.1.tar.gz
+    cd opencv-3.4.1
+    mkdir build
+    cd build
+    cmake -D CMAKE_BUILD_TYPE=RELEASE ..
+    make -j4
+    sudo make install
 
-This package contains some extra nodes and plugins for mavros, please see its [README][uRD].
+# Install klepsydra
 
-installation_instructions
+    rm -rf ~/thirdparties/klepsydra2
+    mkdir -p ~/thirdparties/klepsydra2
+    cd ~/thirdparties/klepsydra2
+    git clone https://github.com/klepsydra-technologies/kpsr-core ~/thirdparties/klepsydra2/kpsr-core
+    cd ~/thirdparties/klepsydra2/kpsr-core
+    git submodule update --init
+
+    cd ~/thirdparties/klepsydra2/
+    git clone https://github.com/klepsydra-technologies/kpsr-robotics ~/thirdparties/klepsydra2/kpsr-robotics
+    cd ~/thirdparties/klepsydra2/kpsr-robotics
+    git submodule update --init
+
+klepsydra core & robotics compilation
+---------------------------------------
+Execute "app_info/kpsr_comp.sh" with -DKPSR_WITH_SOCKET=true
+
+Install boost & valgrind
+------------------------
+    sudo apt install libboost-all-dev
+    sudo apt install valgrind
+
+Install open-license-mngr
+-------------------------
+    cd ~/thirdparties/klepsydra2
+    git clone https://username@repository-open-license-manager.git
+    cd open-license-manager/
+    mkdir build
+    cd build
+    cmake .. -DCMAKE_INSTALL_PREFIX=../install
+    make
+    make install
+    cd ..
+    cd example
+    cmake .
+    make
+
+Generate license procedure
+--------------------------
+Create klepsydra license using generator executable
```

Comment:
remove this block as well... all that is related to the licensing is not really required.